tests/python/topi/python/test_topi_math.py

```diff
@@ -226,8 +228,91 @@ def check_device(device):
     test_apply(topi.fast_tanh, "fast_tanh", np.tanh, low=-10, high=10, step=0.01)
 
 
+_segment_implement = {
+    "segment_max": {
+        "generic": (topi.segment_max, topi.generic.schedule_segment_max),
+    },
+    "segment_min": {
+        "generic": (topi.segment_min, topi.generic.schedule_segment_min),
+    },
+    "segment_mean": {
+        "generic": (topi.segment_mean, topi.generic.schedule_segment_mean),
+    },
+    "segment_sum": {
+        "generic": (topi.segment_sum, topi.generic.schedule_segment_sum),
+    },
+    "segment_prod": {
+        "generic": (topi.segment_prod, topi.generic.schedule_segment_prod),
+    },
+}
+
+
+def verify_segmet(name, data_shape, segmnet_size):
+    def get_segment_ids(length, size):
+        segment_ids = [0]
+        for i in range(size):
+            if np.array(segment_ids).sum() < length:
+                for _ in range(i):
+                    segment_ids.append(i)
+            else:
+                break
+        length = length - len(segment_ids)
+        for i in range(length):
+            segment_ids.append(segment_ids[-1])
+        return np.array(segment_ids).astype("int32")
+
+    def get_ref_data():
```

Comment:
Instead of using tflite to get the reference data, could you add some reference data manually? In this way tests can be independent and also show what the goal of each function is (also, you already added reference tests in `test_forward.py`). What do you think?

Reply:
Okay, thanks for all your comments. All suggestions have been revised.